winml/lib/Api.Ort/OnnxruntimeEngineBuilder.cpp

```diff
@@ -77,4 +82,9 @@ STDMETHODIMP OnnxruntimeEngineBuilder::GetID3D12CommandQueue(ID3D12CommandQueue*
 STDMETHODIMP OnnxruntimeEngineBuilder::SetBatchSizeOverride(uint32_t batch_size_override) {
   batch_size_override_ = batch_size_override;
   return S_OK;
+}
+
+STDMETHODIMP OnnxruntimeEngineBuilder::SetNamedDimensionOverrides(std::map<winrt::hstring, uint32_t> named_dimension_overrides) {
```

Comment:
Is it better to use `ABI::Windows::Foundation::Collections::IMapView` since IEngineBuilder is a COM class?

Reply:
Would be happy to change it but I'm curious what the issue with using STL is? Is it because IEngineBuilder needs to be ABI safe?

Reply:
Switched over to using IMap and IMapView, so iengine.h should be ABI safe now.

Reply:
thanks!